2022-RapidReact/src/main/java/frc/robot/subsystems/StorageSystem.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.VictorSP;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class StorageSystem extends SubsystemBase {
+  /** Creates a new Storage. */
+  private final VictorSP storageConv;
+  public StorageSystem() {
+    storageConv = new VictorSP(Constants.storageConvID);
```

Comment:
Não sei se isso pode ser feito aqui sem ocasionar erros, mas acho que fica mais claro se o motor for declarado e iniciado na mesma linha. Quando vi esse trecho pela primeira vez, fiquei meio confuso.